newIDE/app/src/InstructionOrExpression/SetupInstructionParameters.js

```diff
@@ -65,7 +65,10 @@ export const setupInstructionParameters = (
         );
       });
 
-    if (behaviorNames.length > 0) {
+    if (
+      behaviorNames.length === 1 ||
+      (allowedBehaviorType && behaviorNames.length > 0)
```

Comment:
Add a comment `// Don't auto-select a behavior in the case of allowedBehaviorType being not specified - which in practice is only the case where the user really needs to select a behavior (like the action to enable/disable a behavior). So making a choice there automatically would not make sense.`
and `// In any other case, a specific behavior type is requested, so it make sense to automatically select it (and it might not be shown in the events sheet)`